### ISSUE TITLE
No focus management when routing

### ISSUE THREAD

**Issue by chlane** [vmware/clarity#3173]
## Description  
On https://clarity.design/documentation, after activating any of the router links in the first column on the left, focus remains on the given router link. Screen reader users will not know what happened as a  result of clicking the router link and will not know how to find the newly added content that appeared as a result of activating the router link. 

## Location     
https://clarity.design/documentation

## Priority   
High
  
## Recommended fix      
Focus should be placed in the header of the page it takes you to. See
https://stackblitz.com/github/clane/Angular-A11y-Examples. 

## LOE  
Medium

**Comment by AlexMarcuBytex** [comment]
Can you assign this to me? I m working on a fix.

**Comment by AlexMarcuBytex** [comment]
Can you assign this to me? I m working on a fix.

**Comment by mathisscott** [comment]
@gnomeontherun 
Thoughts on assigning this to @AlexMarcuBytex ?

**Comment by Shijir** [comment]
We are integrating `clrFocusOnViewInit` in vertical navigation for the same kind of purpose. https://github.com/vmware/clarity/pull/3725

The fix to this issue could be simply placing the directive on every page title.

**Comment by AlexMarcuBytex** [comment]
@Shijir cool then. so, will you fix this issue after integration?

**Comment by Shijir** [comment]
closed with #3766
